tasks.py

```diff
@@ -52,7 +52,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_capacity_metrics": {
-            "nautobot_ver": "2.4.20",
+            "nautobot_ver": "next",  # TODO: Change to 3.0.0 after v3.0.0 is released
```

Comment:
This should be changed to 3.0.0 now.

```suggestion
            "nautobot_ver": "3.0.0",
```